docs/source/guides/network_node/staking_guide.rst

```diff
@@ -382,11 +382,27 @@ stake's duration is not shorter than the minimum. To prolong an existing stake's
 Wind Down
 **********
 
-Wind down is *disabled* by default. To start winding down an existing stake:
+Wind down is *disabled* by default.
+When wind down is disabled, the stake's duration remains constant,
+which results in improved staking yield.
+When disabled, the lock duration decreases as time goes by.
```

Comment:
```suggestion
The proportion of staking rewards received by a staker depends on the
stake size and the remaining locked duration. 

When wind down is disabled, the locked duration decreases after each period which results 
in reduced staking yield. When enabled, the stake's locked duration remains 
constant and improves staking yield.

Wind down is *disabled* by default.

```

Reply:
✅

Reply:
Oh actually I just noticed that this suggestion has it backwards wrt to enabled/disabled 😅
I'm going to correct the commit then

Reply:
ah sorry my bad 🤦